docs/Create-Packages/Supporting-Multiple-Target-Frameworks.md

```diff
@@ -46,20 +46,17 @@ In this topic:
 
 ## Framework version folder structure
 
-When building a package that contains only one version of a library, you place its files in the `lib` folder of your project.
-
-When specifically targeting multiple frameworks, you instead make subfolders under `lib` using different case-sensitive framework names with the following convention:
+When building a package that contains only one version of a library or target multiple frameworks, you always make subfolders under `lib` using different case-sensitive framework names with the following convention:
 
     lib\{framework name}[{version}]
 
 For a complete list of supported names, see the [Target Frameworks reference](../schema/target-frameworks.md#supported-frameworks).
 
-If you also have a version of the library that is not specific to a framework, place it in the root `lib` folder.
+You should never have a version of the library that is not specific to a framework, and placed directly in the root `lib` folder (was only supported with NuGet packages.config). Because then it will make it compatible with any target framework and allows it to be installed everywhere but might fail at runtime.
```

Comment:
I would say also that the root is deprecated.
It might also be worth mentioning that sub folders are no longer allowed/deprecated. 
Example `lib/net45/a/a.dll` is ignored in PackageReference

Reply:
updated